pkg/cmd/apply.go

```diff
@@ -17,20 +17,16 @@ limitations under the License.
 package cmd
 
 import (
-	"bufio"
 	"fmt"
 	"os"
 	"path/filepath"
 	"reflect"
-	"strings"
 
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/Masterminds/semver/v3"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
```

Comment:
There should be a new line between those two import groups.

```suggestion
	"github.com/spf13/pflag"

```

Reply:
For some reason, GitHub is not rendering this properly. The new line should be between the `github.com` and `k8c.io` import groups (lines 29-30).